x-team/auto-ui
# Fix z index of hamburger menu

**Release Type:** *Fixes* 


Fixes https://x-team-internal.atlassian.net/browse/XP-2474

## Description

*The Hamburger menu from the top-right hides behind the "Show" button of the Search Form in tabular view*



## Checklist

**Before submitting a pull request,** please make sure the following is done:



## Related PRs
*N/A*


## Steps to Test or Reproduce

1- Go to the Admin page
2- Choose the tabular view
3- click in the hamburger menu
4- You should be able to see the hamburger menu.



## Impacted Areas in Application

Admin page > Tabular view > Hamburger menu



## Screenshots

![fix_haburger_menu_index](https://user-images.githubusercontent.com/5392202/48419453-5d063580-e72e-11e8-916c-47f12eba80e0.gif)
